classes/local/store/object_file_system.php

```diff
@@ -731,11 +731,6 @@ public function presigned_url_should_redirect($contenthash) {
             return false;
         }
 
-        // Redirect regardless.
-        if ($this->externalclient->presignedminfilesize == 0) {
-            return true;
-        }
-
         // Redirect only files that bigger than configured value.
         // And if file extension is not whitelisted.
         if ($this->externalclient->presignedminfilesize > 0) {
```

Comment:
is this if still valid when presignedminfilesize == 0 ??

There should be a fallback return at the end of this function, or the if should be removed

Reply:
yes, the if has be go